oval_graph/command_line_client/client_html_output.py

```diff
@@ -64,6 +67,9 @@ def get_file_name(rule):
 
     def get_save_src(self, rule):
```

Comment:
What could be "save_src"? It looks like that the method returns the filename where to save the output data. Apart from that, the method can create directories, so it's not only getting filenames, but it prepares their directories as well.

The fact that the method modifies the system also affects your ability to test it, I recommend to just output the path, and you can use `pathlib`, e.g. `p.parent.mkdir(parents=True, exist_ok=True)`